recipes/git-pull-request/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "git-pull-request" %}
+{% set version = "6.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6a14c0dc748653833eff3e1822e544f8b5ac45b79b015b643f8805b6d75fc590
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - git-pull-request = git_pull_request:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - attrs
+    - daiquiri
+    - pygithub
+    - python
+
+test:
+  imports:
+    - git_pull_request
+    - git_pull_request.tests
+  commands:
+    - pip check
+    - git-pull-request --help
+  requires:
+    - git
```

Comment:
As it seems you need ``git`` to run this project, you will need to add it to the ``run`` requirements

Reply:
yep, did add it :smile:

Reply:
```suggestion
```

Reply:
I still think we should remove git from the tests and add it to the ``run`` requirements, is that a problem?

Reply:
Adding it to `run` is indeed better, and if it is not needed for the tests, then it should be removed. I updated, let's see what the checks say.

Reply:
they agree :-)